app/services/nytimes_service.rb

```diff
@@ -10,6 +10,11 @@ def articles
     response_body = parse(connection.get)
     articles_data = response_body["results"]
     Article.create_articles(articles_data)
+    keyword_engine = KeywordEngine.new
+    Article.all.each do |article|
+      keywords = keyword_engine.generate_keywords(article)
+      article.update(keywords: keywords)
+    end
```

Comment:
We should work on refactoring this soon. Maybe we can create a notes.txt to make simple reminders..